thirdparty/games_are_not_uwp.patch

```diff
@@ -0,0 +1,57 @@
+diff --git a/thirdparty/README.md b/thirdparty/README.md
+index dd5184a5ce..c4be92d171 100644
+--- a/thirdparty/README.md
++++ b/thirdparty/README.md
+@@ -276,6 +276,8 @@ Files extracted from upstream source:
+ - the `src` folder
+ - `AUTHORS`, `COPYING`, `THANKS`
+ 
++Applied the patch in `patches/games_are_not_uwp.patch` (upstream PR: https://github.com/harfbuzz/harfbuzz/pull/4109).
++
+ 
+ ## icu4c
+ 
+diff --git a/thirdparty/harfbuzz/src/hb-blob.cc b/thirdparty/harfbuzz/src/hb-blob.cc
+index f0fda1fa4d..265effba03 100644
+--- a/thirdparty/harfbuzz/src/hb-blob.cc
++++ b/thirdparty/harfbuzz/src/hb-blob.cc
+@@ -676,7 +676,7 @@ fail_without_close:
+   wchar_t * wchar_file_name = (wchar_t *) hb_malloc (sizeof (wchar_t) * size);
+   if (unlikely (!wchar_file_name)) goto fail_without_close;
+   mbstowcs (wchar_file_name, file_name, size);
+-#if !WINAPI_FAMILY_PARTITION(WINAPI_PARTITION_DESKTOP)
++#if !WINAPI_FAMILY_PARTITION(WINAPI_PARTITION_DESKTOP) && WINAPI_FAMILY_PARTITION(WINAPI_PARTITION_APP)
+   {
+     CREATEFILE2_EXTENDED_PARAMETERS ceparams = { 0 };
+     ceparams.dwSize = sizeof(CREATEFILE2_EXTENDED_PARAMETERS);
+@@ -697,7 +697,7 @@ fail_without_close:
+ 
+   if (unlikely (fd == INVALID_HANDLE_VALUE)) goto fail_without_close;
+ 
+-#if !WINAPI_FAMILY_PARTITION(WINAPI_PARTITION_DESKTOP)
++#if !WINAPI_FAMILY_PARTITION(WINAPI_PARTITION_DESKTOP) && WINAPI_FAMILY_PARTITION(WINAPI_PARTITION_APP)
+   {
+     LARGE_INTEGER length;
+     GetFileSizeEx (fd, &length);
+@@ -710,7 +710,7 @@ fail_without_close:
+ #endif
+   if (unlikely (!file->mapping)) goto fail;
+ 
+-#if !WINAPI_FAMILY_PARTITION(WINAPI_PARTITION_DESKTOP)
++#if !WINAPI_FAMILY_PARTITION(WINAPI_PARTITION_DESKTOP) && WINAPI_FAMILY_PARTITION(WINAPI_PARTITION_APP)
+   file->contents = (char *) MapViewOfFileFromApp (file->mapping, FILE_MAP_READ, 0, 0);
+ #else
+   file->contents = (char *) MapViewOfFile (file->mapping, FILE_MAP_READ, 0, 0, 0);
+diff --git a/thirdparty/libwebp/src/utils/thread_utils.c b/thirdparty/libwebp/src/utils/thread_utils.c
```

Comment:
Probably should be separate patches for HarfBuzz and WebP.

Reply:
WebP was not really needed. Removed in upcoming push.